astroquery/utils/tap/taputils.py

```diff
@@ -255,22 +256,22 @@ def get_suitable_output_file_name_for_current_output_format(output_file, output_
         'fits', 'csv', 'ecsv' and 'json'. Default is 'votable'.
         Returned results for formats 'votable' 'ecsv' and 'fits' are compressed
         gzip files.
+    format_with_results_compressed: a set of output formats
```

Comment:
sphinx requires the whitespaces around `:` to render the parameters correctly
```suggestion
    format_with_results_compressed : tuple of str, optional
        a set of output formats
```